### DISCUSSION TITLE
Installing InputLeap where Barrier IS/WAS

### DISCUSSION THREAD

**Discussion by Supastyles** [input-leap/input-leap#2129] [Q&A]
I went through ther readme and searched discussions and wasn't able to see if this has been mentioned(if it is Im sorry I missed it)  

So I've been using Barrier for a while and I kind of forgot it was an old program/version and around the time I installed it I was still fairly new to github (occasionally 'used' it over the years but hadn't really got into the nuts and bolts of how everything actually worked)  so I wasn't really aware of what forks and branches were etc.  I've since been expanding my horizons etc and revisited Barrier then came accross deskflow...  Again been not REALLY familiar I read a little and assumed that felt like the natural decendant of Barrier..  So I attempted to install and I ran into some issues and wasn't able to get it to function.  Back to their page and Barrier and I came accross a wiki that discusses those as well as InputLeap and Synergy(I think)  reading about InputLeap I assumed it was more of a parralel brank to deskflow but I reallly couldn't tell what the differences in the 2 were (other then some mentions of back end differences) but they seemed to describe the same functionality.  So D/L'd input Leap and tried installing that... This is where things started to go really wrong on my end.  (my fault Im sure)  I feel like I now had 3 similar overlapping versions of the same but different software installed and this is where I noticed there was some co dependecies(? not sure if that's the right term) or shared file and folder destinations.  At this point I had all 3 installed and kind of running and funtionality remained but there were issues.  I started stopping some app windows(closing) then I lost functionality and then couldn't get it back(plus I hit the bug that I've since read about in the Issues section withe Bonjour not being recognised or installed.  so I started backing out of the mess by uninstallling items.  First DF.. then IL then Barrier but having ruined it with the other two it was kind of stuck (glad I had used REVO installer when I initially added it) I couldn;t reinstall Barrier (2.4) because the barrierc.exe was locked up and there was nothing in the tasks or services I could see that was holding onto it and used file unlocker and no success finally was able to fix it with the REVO uninstalled to clear out alll the old files..  

Long story short.  What is the reconmondation when switching to InputLeap?  should we fully remove Barrier and keep the 3 branches seperated within the systems?  Or did I just mess it all up somhow on my end?

.. FYI I've read about the compatibility between the 3 which seems more like a system to system compatibility that currently exists as a byproduct of the applications origins and not as a intended use case.   
If I am right in assuming that maybe they should be removed prior perhaps building that into the installation file(s) and or in the meantime a short instruction in the readme or release notes.  

Sorry it was so long!

**Comment by shymega** [comment]
This really doesn't make a lot of sense. I don't really have much time left nowadays, so if you can please clarify in shorter, composable and logical paragraphs, that would be of use.
